Ingress/images/nginx-slim/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM alpine:3.2
+
+RUN apk add --update-cache bash && \
+  rm -rf /var/cache/apk/*
+
+COPY build.sh /tmp
+
+RUN /tmp/build.sh
+
+# forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log
+RUN ln -sf /dev/stderr /var/log/nginx/error.log
```

Comment:
If I wrap another program that logs around this, that programs logs will show up in docker logs right? (feel like that will work out, just confirming)